### ISSUE TITLE
Problema con los CORS

### ISSUE THREAD

**Issue by IJuanjo** [qonteo/reactjs#3]
intente usar la api para el login pero logra botarme un error con los cors bloqueados, lo estoy consumiendo por javascript

Access to fetch at 'https://dashboard.qonteo.com/REST/login' from origin 'http://localhost' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

**Comment by qonteo** [comment]
"Access-Control-Allow-Origin" “*” has been added to the headers of https://dashboard.qonteo.com/REST/login <https://dashboard.qonteo.com/REST/login>

Please, review and let us know if the error related to CORS still remains.

p.s. Probably you have to disable CORS in your end as well.

**Comment by IJuanjo** [comment]
I require information about the data that the api brings, I see that it only brings a few results

**Comment by qonteo** [comment]
Moved to another Issue  https://github.com/qonteo/reactjs/issues/4
